Mathlib/GroupTheory/GroupAction/Basic.lean

```diff
@@ -411,6 +411,10 @@ theorem orbitRel_apply {a b : α} : (orbitRel G α).Rel a b ↔ a ∈ orbit G b
 #align mul_action.orbit_rel_apply MulAction.orbitRel_apply
 #align add_action.orbit_rel_apply AddAction.orbitRel_apply
 
+@[to_additive]
+lemma orbitRel_r_apply {a b : α} : @Setoid.r _ (orbitRel G _) a b ↔ a ∈ orbit G b :=
```

Comment:
I believe the following works
```suggestion
lemma orbitRel_r_apply {a b : α} : (orbitRel G _).r a b ↔ a ∈ orbit G b :=
```
since Lean 4 tries to fill dot notation arguments to any kind of parameters, not just explicit